### ISSUE TITLE
On transplantation

### ISSUE THREAD

**Issue by Xiaojin666666** [Charles445/LaMulana2OldMusic#1]
I would like to discuss with you the porting of Minecraft component in China. Are you interested? How can I contact you

**Comment by xiaojinChina** [comment]
I would like to discuss with you the porting of my Mineccarft component in China. Are you interested? How can I contact you

**Comment by Charles445** [comment]
Are you interested in hosting the minecraft mods on other websites? If so, then that is okay, you have my permission to host them.

**Comment by Xiaojin666666** [comment]
Yes brother, but do you need the copyright fee to distribute your works? I come from China, if you are willing, I will transplant your components, develop and release them to the Chinese game platform to let more people know, I want to transplant your [more Enchantment book].



---- Replied Message ----
| From | ***@***.***> |
| Date | 08/16/2022 21:37 |
| To | ***@***.***> |
| Cc | ***@***.***>、State ***@***.***> |
| Subject | Re: [Charles445/LaMulana2OldMusic] On transplantation (Issue #1) |

Are you interested in hosting the minecraft mods on other websites? If so, then that is okay, you have my permission to host them.
